src/main.rs

```diff
@@ -218,6 +218,9 @@ struct Cli {
     #[arg(long, default_value = "none")]
     /// Show call debug information
     show_calls: ShowCalls,
+    #[arg[long]]
+    /// Show call output
+    show_outputs: bool,
```

Comment:
This is awesome, thank you!!

One thing we've done with the other feature flags, is to have them be configurable via API calls in the `config_` namespace( example [config_setResolveHashes](https://github.com/matter-labs/era-test-node/blob/main/src/node/config.rs#L153)). So if you forgot to enable Call Output when you start the in-memory node, you don't have to restart the node and potentially lose state.

Here are some other `config_` API examples: https://github.com/matter-labs/era-test-node/blob/main/SUPPORTED_APIS.md#config_setresolvehashes

Reply:
That makes sense, added the API calls